twa/app/src/main/res/xml/shortcuts.xml

```diff
@@ -1,16 +1 @@
-<!--
```

Comment:
Please undo this change.

Reply:
@saghul 
Please check again. Thank you.